### PR TITLE
Xode PR Arc8_2B

### PR DESCRIPTION
**Pull request by Xode672**

### CODE REVIEW

File: script/Arcueid/Day 8/08_00_ARC08_2B.txt [hunk 1]
```diff
@@ -112,7 +112,7 @@ He's right.
 -- ここは遠野邸前の国道で、通るのは乗用車ぐらい。
 // 国道 is decidedly not a western conception of 'highway'
 We're on the road in front of the Tohno residence,
- and the only traffic it usually gets is the odd car.
+ and the only traffic it usually gets is the odd car. //What does it mean by Odd car?
```
Comment (yk1512) [inline script/Arcueid/Day 8/08_00_ARC08_2B.txt:115]: It means that there is not much traffic, only cars pass every so often. Its a standard phrase.

File: script/Arcueid/Day 8/08_00_ARC08_2B.txt [hunk 2]
```diff
@@ -157,7 +157,7 @@ It's more that I'm terrified by how intent he is.
 -- なんでもない言葉の中に、こちらの態度を一つも取りこぼさないよう注目する<本|・><気|・>を感じたというか……。
 I get the impression that he's paying close attention to my every word,
  no matter how trifling,
- careful not to miss any clues as to my attitude.
+ careful not to miss any clues as to my attitude.//Is he saying that Ando is taking heavy note on Shiki's attitude or that Ando taking note on everything about Shiki including his attitude?
```
Comment (yk1512) [inline script/Arcueid/Day 8/08_00_ARC08_2B.txt:160]: he's closely monitoring his attitude.

File: script/Arcueid/Day 8/08_00_ARC08_2B.txt [hunk 3]
```diff
@@ -172,7 +172,7 @@ Well, maybe I am, but I'll make this worth your while, okay?
 [sha:084f732393fb8cea54b9ae2cde4f90e0a1fd7051]{
 -- Page 104, Offset 19929. Mods: @k, @e.
 -- とくに知りたいのはほら、最近帰ってきたっていう長男の話。来たばっかりで写真資料もなくてさあ。どんな外見なのか、好きな食べ物とか好きな女の子のタイプとか、ぶっちゃけ何に弱いのか教え―――
-What I really want to know about is the guy that just returned to the family, y'know, the eldest son.
+What I really want to know about is the guy who'd recently returned to the family, y'know, the eldest son.
```
Comment (yk1512) [inline script/Arcueid/Day 8/08_00_ARC08_2B.txt:175]: revert imo

File: script/Arcueid/Day 8/08_00_ARC08_2B.txt [hunk 4]
```diff
@@ -222,7 +222,7 @@ But he doesn't smell like a bourgeois at all, does he?!"
 "What kind of smell is that? It's not like you can smell money, right?
  Or are you talking about the smell of the filth on his hands?
  Are you saying that's all the wealthy are? Filth?
- Those sound like fighting words to me, oi".
+ Those sound like fighting words to me, oi".//Are we using "Fighting Words" like the legal term? Tldr: Words with intent to provoke violence/hatred from the target, that cannot be protected/justified under "Freedom of Speech"
```
Comment (yk1512) [inline script/Arcueid/Day 8/08_00_ARC08_2B.txt:225]: We are not. He's just commenting on the provocative attitude

**Review by yk1512 — CHANGES_REQUESTED** [review]

File: script/Arcueid/Day 8/08_00_ARC08_2B.txt [hunk 5]
```diff
@@ -238,12 +238,12 @@ But he doesn't smell like a bourgeois at all, does he?!"
 [sha:d2dd8340f9831edc527793ef739d937bce5e6789]{
 -- Page 109, Offset 19940.
 -- ……なんなんだ、この状況……。
-...What on earth have I stumbled into here...
+...What on earth have I been caught up in here...
```
Comment (rschlaikjer) [inline script/Arcueid/Day 8/08_00_ARC08_2B.txt:241]: Prefer original, it doesn't feel like he's really caught up in anything/ involved yet, just observing

File: script/Arcueid/Day 8/08_00_ARC08_2B.txt [hunk 6]
```diff
@@ -238,12 +238,12 @@ But he doesn't smell like a bourgeois at all, does he?!"
...
 }
 [sha:55d7f85f6793a819b228c0b9cf91af2240040d8b]{
 -- Page 109, Offset 19941.
 -- とりあえず警察を呼ぶべきか？
-Maybe I should call the cops and let them deal with it?
+Maybe I should just call the cops and let them deal with it?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 8/08_00_ARC08_2B.txt:246]: `just` gets a lot of use, including _twice_ on the next line. Maybe leave it off here or find something else, unless it really feels necessary for flow

File: script/Arcueid/Day 8/08_00_ARC08_2B.txt [hunk 7]
```diff
@@ -319,7 +319,7 @@ These have to be the two most suspicious detectives I've ever met.
 -- Page 114, Offset 19955.
 -- あの手帳が本物かどうかはともかく、うちの前で張り込んでいた理由は気にかかる。
 Regardless of whether or not that ID card is the real deal,
- I'm curious why they're staking out the area in front of our house.
+ I'm curious as to why they're staking out the area in front of our house.
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_2B.txt:322]: revert

**Review by Avuxo — APPROVED** [review]

**Pull request merged**